### PR TITLE
Added Driver Edit and Create Page + Tests

### PR DESCRIPTION
**Pull request by Aasish-Virjala**
In this PR, we added the Edit and Create Pages. Screenshots attached below.

Screenshot 2023-08-27 at 7 30 22 PM
Screenshot 2023-08-27 at 7 30 37 PM

### CODE REVIEW

File: frontend/src/main/components/Shift/ShiftForm.js [hunk 1]
```diff
@@ -62,7 +63,33 @@ function ShiftForm({ initialContents, submitAction, buttonLabel = "Create" }) {
                     {errors.day?.message}
                 </Form.Control.Feedback>
             </Form.Group>
+ */}
```
Comment (pconrad) [inline frontend/src/main/components/Shift/ShiftForm.js:66]: commented out code?

**Review by pconrad — CHANGES_REQUESTED** [review]
Apart from the commented out code, LGTM.  Fix that, and we can get this merged

File: frontend/src/main/components/Shift/ShiftForm.js [hunk 2]
```diff
@@ -38,6 +38,7 @@ function ShiftForm({ initialContents, submitAction, buttonLabel = "Create" }) {
                 </Form.Group>
             )}
 
+{/*
```
Comment (pabucsb) [inline frontend/src/main/components/Shift/ShiftForm.js:41]:
```suggestion
```

File: frontend/src/main/components/Shift/ShiftForm.js [hunk 3]
```diff
@@ -38,6 +38,7 @@ function ShiftForm({ initialContents, submitAction, buttonLabel = "Create" }) {
                 </Form.Group>
             )}
 
+{/* 
             <Form.Group className="mb-3" >
```
Comment (pabucsb) [inline frontend/src/main/components/Shift/ShiftForm.js:42]:
```suggestion
  
```

File: frontend/src/main/components/Shift/ShiftForm.js [hunk 4]
```diff
@@ -38,6 +38,7 @@ function ShiftForm({ initialContents, submitAction, buttonLabel = "Create" }) {
...
                 <Form.Label htmlFor="day">Day of Week</Form.Label>
```
Comment (pabucsb) [inline frontend/src/main/components/Shift/ShiftForm.js:43]:
```suggestion

```

File: frontend/src/main/components/Shift/ShiftForm.js [hunk 5]
```diff
@@ -38,6 +38,7 @@ function ShiftForm({ initialContents, submitAction, buttonLabel = "Create" }) {
...
                 <Form.Select
```
Comment (pabucsb) [inline frontend/src/main/components/Shift/ShiftForm.js:44]:
```suggestion
```

File: frontend/src/main/components/Shift/ShiftForm.js (hunk 3 above)
Comment (pabucsb) [inline frontend/src/main/components/Shift/ShiftForm.js:42]:
```suggestion

```

File: frontend/src/main/components/Shift/ShiftForm.js [hunk 6]
```diff
@@ -62,7 +62,33 @@ function ShiftForm({ initialContents, submitAction, buttonLabel = "Create" }) {
                     {errors.day?.message}
                 </Form.Control.Feedback>
             </Form.Group>
+ */}
```
Comment (pabucsb) [inline frontend/src/main/components/Shift/ShiftForm.js:65]:
```suggestion

```

**Review by pconrad — APPROVED** [review]
LGTM!

**Pull request merged**